documentation/3.4.0/setup/docker/samiksha-with-project/windows/README.md

```
@@ -0,0 +1,323 @@
+    echo Port %p is available
+  ) else (
+    echo Port %p is IN USE
+  )
+)
+```
+
+1.  **Download Docker Compose File:** Retrieve the **[docker-compose-samiksha.yml](https://github.com/ELEVATE-Project/project-service/raw/main/documentation/1.0.0/dockerized/docker-compose-samiksha.yml)** file from the samiksha service repository and save it to the samiksha directory.
+
+    ```
+    curl -OJL https://raw.githubusercontent.com/ELEVATE-Project/project-service/refs/heads/main/documentation/3.4.0/dockerized/dockerFiles/project-with-survey/docker-compose-project.yml
+    ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)